Sigint fix windows

krisdehaudt opened the pull request:
This fix handles the problem about the SIGINT signal that is not known
in Windows because this is a signal specific for Linux. This can be
fixed by changing the signal to 'exit'. (I have also reverted the commit about the configuration file here)

By my opinion, it is best that you make an extra branch for Windows because it is specific for Windows. I think this fix will create a problem in Linux. (but I'm not sure)

Commit:
Fix bug about the configuration file

This is a fix for a problem with the configuration file. When autolint
is started for the first time, it will generate a file named
autolint.js. But when for the second time the autolint command is
called, it will try to execute the javascript-file instead of the
command. This is fixed by changing the name of the configuration file so
it doesn't match anymore with the name of the command. Here is the
configuration file called autolintconfig.js.

Commit:
Windows fix about SIGINT signal

This fix handles the problem about the SIGINT signal that is not known
in Windows because this is a signal specific for Linux. This can be
fixed by changing the signal to 'exit'.

magnars commented:
This only shifts the problem from windows over to linux. I'm not willing to maintain two versions of autolint. This is just sloppy. If you're not willing to put in the work to get it working on both OSes, that's fine. I'll take a look at getting in a proper fix after my vacation.

krisdehaudt commented:
Ok, I will try to look in the weekend for a solution because I gave more time then.

Edit: I fixed so it works on both the platforms. It is maybe not the most beautifull solution but it works.
Edit2: I implemented a better fix by using something I found when looking for different things. (My excuses for the many commits and comments)

Commit:
Fix about SIGINT

This fix works both for Windows and Linux

Commit:
Better fix about SIGINT and Windows

magnars commented:
Fixed in 6bc2707. Thanks.
